src/config/settings/rest_framework.py

```diff
@@ -47,4 +47,13 @@
     "TOKEN_BLACKLIST_SERIALIZER": "rest_framework_simplejwt.serializers.TokenBlacklistSerializer",
     "SLIDING_TOKEN_OBTAIN_SERIALIZER": "rest_framework_simplejwt.serializers.TokenObtainSlidingSerializer",
     "SLIDING_TOKEN_REFRESH_SERIALIZER": "rest_framework_simplejwt.serializers.TokenRefreshSlidingSerializer",
-}
\ No newline at end of file
+}
+
+
+#Settings for connection to the gmail account
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'anynoteconfirm@gmail.com'
+EMAIL_HOST_PASSWORD = 'mcqb jsgo zvki aznt'
```

Comment:
hardcoded password not in local settings
I would suggest to put it in local settings files and set the exemplar one in template settings also

Reply:
It is done